tests/general/CheckmanifestBearTest.py

```diff
@@ -0,0 +1,77 @@
+import shutil
+import tempfile
+from os import path
+import unittest
+
+from queue import Queue
+
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from bears.general.CheckmanifestBear import CheckmanifestBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+
+class CheckmanifestBearTest(unittest.TestCase):
+
+    def setUp(self):
+        self.test_dir = tempfile.mkdtemp()
+        self.testt_dir = tempfile.mkdtemp()
+        self.queue = Queue()
+        self.file_dict = {}
+        self.section = Section('test')
+        self.section.add_or_create_setting(Setting('ignore', 'unrelated.txt'))
+        self.uut = CheckmanifestBear(self.file_dict, self.section,
+                                     self.queue)
+
+    def tearDown(self):
+        shutil.rmtree(self.test_dir)
+        shutil.rmtree(self.testt_dir)
+
+    def run_uut(self, *args, **kwargs):
+        return list(result.message for result in self.uut.run())
```

Comment:
change to `return list(result.message for result in self.uut.run(*args, **kwargs))`